src/content/tools/devtools/network.md

```diff
@@ -55,6 +55,20 @@ my-endpoint m:get t:json s:200
 https s:404
 ```
 
+### Recording network requests on app startup
+
+To record network traffic on app startup, you can start your app in a paused state then, begin recording before your app starts:
```

Comment:
"... state, and then begin recording network traffic in DevTools before resuming your app."

Reply:
Thanks, @kenzieschmoll. Also, @CoderDake, can you insert a line feed after the comma? We break our lines at 80 chars or less.

And thanks for this PR!

Reply:
ty  for noticing this, Looks like I accidentally had word wraps turned on in my IDE so I didn't notice the line break issues.